DEVELOPER_GUIDE.md

```diff
@@ -53,6 +54,16 @@ opensearch-plugin install file:///path/to/target/releases/opensearch-job-schedul
 ```
 to install the JobScheduler plugin to your OpenSearch.
 
+## To Publish generated plugin zips to maven repo
+The generated plugin zip `opensearch-job-scheduler-<version>.zip` after the build, inside the folder `build/distributions` can be published to maven repo. This can be done by including the project `pluginZips` inside `settings.gradle` file. Once added the generated plugin zip will be published to maven repo with following maven coordinates
+```
+  <groupId>org.opensearch.plugin</groupId>
+  <artifactId>opensearch-security</artifactId>
```

Comment:
This should be job scheduler not security, right?

Reply:
Thanks I have updated in my latest commit.